src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -226,13 +225,33 @@ bool AlertManeuverRequest::IsWhiteSpaceExist() {
     smart_objects::SmartArray::const_iterator it_tc_end = tc_array->end();
 
     for (; it_tc != it_tc_end; ++it_tc) {
-      str = (*it_tc)[strings::text].asCharArray();
+      const char* str = (*it_tc)[strings::text].asCharArray();
       if (strlen(str) && !CheckSyntax(str)) {
         LOG4CXX_ERROR(logger_, "Invalid tts_chunks syntax check failed");
         return true;
       }
     }
   }
+  if ((*message_)[strings::msg_params].keyExists(strings::soft_buttons)) {
+    DCHECK_OR_RETURN(
```

Comment:
@LuxoftAKutsan maybe it will be better to replace `DCHECK` with `LOG4CXX_ERROR + return true` as it's done in lines 249-250? I don't think that wrong type in rpc message is a reason for fatal error and SDL termination.